README.md

```diff
@@ -67,7 +67,7 @@ You can use the following command to install MagTape and the example policies fr
 **NOTE:** The `master` branch of this repository is considered a working branch and may not always be in a functioning state. It's best to select a specific tag for a stable version of MagTape
 
 ```
-$ kubectl apply -f https://raw.githubusercontent.com/tmobile/magtape/v2.3.2/deploy/install.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/tmobile/magtape/v2.3.3/deploy/install.yaml
```

Comment:
**question (non-blocking)**: when looking into md style it seems that common practice is terminal/shell examples which do not show the anticipated output should not include a leading $. Do we want to consider adjusting magtape md style? I think magtape is pretty consistent internally so not a big deal either way just thought I'd mention it.

Reply:
It's something we can discuss offline. It's not my preference, idiomatic or not, but I'm open to discussing it if there are strong feelings.